app/PharBuilder.php

```diff
@@ -353,7 +353,7 @@ public function buildPhar()
 
         $this->stubFile = $this->makePathRelative($this->stubFile);
         $this->phar->setStub(
-            (!$this->isSkipShebang() ? '#!/usr/bin/env php' . PHP_EOL : '') .
+            (!$this->isSkipShebang() ? '#!/usr/bin/env php' . "\n" : '') .
```

Comment:
May be it's obvious for those who created shell scripts at least once? If you want, I can add comment, but think it's unnecessary